.github/workflows/tests.yml

```diff
@@ -35,12 +35,54 @@ jobs:
       - name: Coverage reporting
         run: |
           coverage combine
-          coverage report -m
-          coverage xml
-          coverage html
```

Comment:
I'm not sure I understand why coverage xml and html files were produced. xml, I'd guess it's because codecov needs it (though I would have thought it used the .coverage file itself).
.html, I really have no idea. Of course, if this serves a purpose that I didn't see, please tell me, I'll be delighted to re-add it with a comment to explain why.

Reply:
This was a legacy thing. (Don't ask 😅) Fine to get rid of it.